types/tlds/tslint.json

```diff
@@ -0,0 +1,8 @@
+{
+    "extends": "dtslint/dt.json",
+    "rules": {
+        "no-empty-interface": false,
```

Comment:
For new packages, this file should only `"extends": "dtslint/dt.json"`, you can disable some rules in source code like `// tslint:disable-next-line no-empty-interface`